docs/starters.yml

```diff
@@ -90,6 +90,14 @@
     - Includes plugins for easy, beautiful typography
     - Includes React Helmet to allow editing site meta tags
     - Includes plugins for offline support out of the box
+- url: https://github.com/gatsbyjs/gatsby/tree/master/starters/gatsby-starter-blog-theme
+  repo: https://github.com/gatsbyjs/gatsby/tree/master/starters/gatsby-starter-blog-theme
+  description: official blog theme
+  tags:
+    - Official
+    - Blog
+  features:
+    - Includes plugin for adjustment of background lighting
```

Comment:
I think there are more features we could add here. The blog theme has:
- blog posts set up from a content folder
- styling done with Theme-UI showing how to shadow a theme file

...and more stuff set up for you. It would be good to highlight what we can, if this is really to help showcase these official starters we should make them shine!